source/server/spongineer/incompatible.rst

```diff
@@ -14,109 +14,139 @@ To help you resolve which is which, and how to fix the problem (if possible), we
+- Github issue: `SpongeForge 1.16.5 InvalidMixinException ATM6 <https://github.com/SpongePowered/Sponge/issues/3647>`_
+
+Applied Energistics 2
+~~~~~~~~~~~~~~~~~~~~~
+
+- Versions: 8.4.7
+- Problem: Non-fatal console error when opening `Drive` gui
+- Solution: None so far
+- Github issue: `Logged slot transactions without event <https://github.com/SpongePowered/Sponge/issues/3680>`_
+
+Bountiful Baubles
+~~~~~~~~~~~~~~~~~
+
+- Version: 1.16.5-0.1.0-forge
+- Problem: Crash on startup
+- Solution: None so far
+- Github issue: `Bountiful Baubles blows up SF 1.16.5 <https://github.com/SpongePowered/Sponge/issues/3646>`_
+
+Conquest Reforged
+~~~~~~~~~~~~~~~~~
+
+- Version: 5.0.7
+- Problem: Breaking a block with a snow layer in hand causes crash
+- Solution: None so far
+- Github issue: `Server crashes when creaking a block with snow layer in your hand <https://github.com/SpongePowered/Sponge/issues/3621>`_
+
+Create Mod
+~~~~~~~~~~
+
+- Version: mc1.16.5_v0.3.2g
+- Problem: Non-fatal console error when opening an inventory
+- Solution: None so far
+- Github issue: `Logged slot transactions without event <https://github.com/SpongePowered/Sponge/issues/3680>`_
+
+Pehkui
+~~~~~~
+
+- Version: 3.4.2
 - Problem: Crash on startup
-- Solution: Change ``B:useconfigs=false`` to ``true`` in the ``config/mystcraft/core.cfg`` file
-
-Open Terrain Generator (OTG)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-- Dependencies: OTGcore
-- Versions: All (so far)
-- Problem: Crash on startup / Multiworld Wgen problems
-- Solution: Pre-generate world without SpongeForge, then remove OTG and add SpongeForge.
-
-Phosphor
-~~~~~~~~
-- Versions: Up to 0.2.4
-- Problem (1): Crash on startup
-- Solution: Update to version 0.2.5 or newer, which is compatible with Sponge RC3844.
-- Problem (2): Poor graphic performance
-- Solution: Set the optimisation ``async-lighting`` to ``false`` in the Sponge ``global.conf`` file.
-
-There may be many more, please help us keep this list maintained by contributing to the SpongeDocs on GitHub!
-The :doc:`debugging` page may also be of help if your issue is not one of those mentioned above.
+- Solution: None so far
+- Github issue: `SpongeForge incompatibility with pehkui <https://github.com/SpongePowered/Sponge/issues/3829>`_
+
+Physics Mod
+~~~~~~~~~~~
+
+- Version: 1.3.4
+- Problem: Lag on TNT use 
+- Solution: None so far
+- Github issue: `Fatel server lag when using TNT and Physics mod <https://github.com/SpongePowered/Sponge/issues/3517>`_
+
+Progressive Bosses
+~~~~~~~~~~~~~~~~~~
+
+- Version 3.4.3
+- Problem: Crash on startup
+- Solution: Update SpongeForge to the latest
+- Github issue: `Illegal State Exception with Progressive Bosses mod <https://github.com/SpongePowered/Sponge/issues/3714>`_
+
+Random patches
+~~~~~~~~~~~~~~
+
+- Version: 2.4.4
+- Problem: Crash on startup
+- Solution: Update SpongeForge to the latest
+- Github issue: `Server crash with Randompatches <https://github.com/SpongePowered/Sponge/issues/3589>`_
+
+Tickers' Construct
```

Comment:
It's called `Tinkers' Construct`. (The apostrophe is optional, according to the mod page...)

Reply:
Updated